kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -242,6 +242,7 @@
     }
   },
   "variables": {
+    "apiVersionDeployments": "2018-06-01",
```

Comment:
Can you check again what version we need to use because as per my understanding it should be 2015-11-01.

Reply:
the version used in AKS engine is 2018-06-01, I tested it and decided to use that to keep things consistent